pkg/apis/policy/v1alpha1/clusterimagepolicy_types.go

```diff
@@ -163,6 +168,15 @@ type TLog struct {
 	URL *apis.URL `json:"url,omitempty"`
 }
 
+// RFC3161Timestamp specifies the URL to a RFC3161 time-stamping server that holds
+// the time-stamped verification for the signature
+type RFC3161Timestamp struct {
+	// TODO: Rename this to intermediate certificate only...
+	// CertChain sets a reference to certificate chain used for verification
+	// +optional
+	CertChain *KeyRef `json:"cert-chain,omitempty"`
```

Comment:
Maybe this should be a TrustRootRef instead of baking the CertChain here? The TrustRootRef for TimeStampAuthorities has all the information I believe that is needed for verification?

Reply:
Yes, I'll make this change now.

Reply:
I am gonna use. There are many other things inside the TrustRootRef that are not related and shouldn't be exposed within the RFC3161Timestamp.

```
// RFC3161Timestamp specifies the URL to a RFC3161 time-stamping server that holds
// the time-stamped verification for the signature
type RFC3161Timestamp struct {
	// Trusted timestamping authorities
	TimeStampAuthorities []config.CertificateAuthority `json:"timestampAuthorities"`
}

```

Reply:
I am having second thoughts about what I just said above. I'll use a TrustRootRef but only rely on the list of TimeStampAuthorities.